src/libraries/System.Resources.Extensions/src/System.Resources.Extensions.csproj

```diff
@@ -59,32 +59,30 @@ System.Resources.Extensions.PreserializedResourceWriter</PackageDescription>
   </ItemGroup>
 
   <PropertyGroup>
-    <_packageTargetsFile>$(IntermediateOutputPath)$(AssemblyName).targets</_packageTargetsFile>
   </PropertyGroup>
 
   <Target Name="GeneratePackageTargetsFile" 
-          Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'"
```

Comment:
This is going to run for each inner TFM now. Is that intentional?

Reply:
No, BeforePack is an outer build hook which runs only once.